examples/networkfirewall/firewallpolicy-complex.yaml

```
@@ -0,0 +1,124 @@
+apiVersion: networkfirewall.aws.upbound.io/v1beta1
+kind: FirewallPolicy
+metadata:
+  annotations:
+    meta.upbound.io/example-id: networkfirewall/v1beta1/firewallpolicy-complex
```

The example ID specifies a GVK and is important so that we can correctly show the example in the marketplace:
```suggestion
    meta.upbound.io/example-id: networkfirewall/v1beta1/firewallpolicy
```